### PR TITLE
Dark Mode: Nivo Charts - RSC-1660

### PR DESCRIPTION
**Pull request by rpokorny**
https://issues.sonatype.org/browse/RSC-1660

### CODE REVIEW

File: lib/src/base-styles/_nx-colors.scss [hunk 1]
```diff
@@ -289,7 +289,15 @@
   --nx-color-system-notice-text: var(--nx-color-text);
 
   --nx-color-chart-line: var(--nx-swatch-grey-40);
-  --nx-color-chart-text: var(--nx-color-text-dark);
+  --nx-color-chart-text: var(--nx-color-text-stark);
+  --nx-color-chart-data-1: var(--nx-swatch-teal-40);
+  --nx-color-chart-data-2: var(--nx-swatch-red-65);
+  --nx-color-chart-data-3: var(--nx-swatch-green-35);
```
Comment (avnvbhatta) [inline lib/src/base-styles/_nx-colors.scss:295]: In Kaleido, the the green is a `green-80`.

File: lib/src/util/nivo.tsx [hunk 2]
```diff
@@ -6,14 +6,14 @@
...
-  'var(--nx-swatch-turquoise-30)',
-  'var(--nx-swatch-indigo-60)'
+  'var(--nx-color-chart-data-1)',
+  'var(--nx-color-chart-data-2)',
+  'var(--nx-color-chart-data-3)',
+  'var(--nx-color-chart-data-4)',
+  'var(--nx-color-chart-data-5)',
+  'var(--nx-color-chart-data-6)',
+  'var(--nx-color-chart-data-7)',
+  'var(--nx-color-chart-data-8)'
```
Comment (avnvbhatta) [inline lib/src/util/nivo.tsx:16]: Needs to be documented.

File: lib/src/base-styles/_nx-colors.scss (hunk 1 above)
Comment (avnvbhatta) [reply lib/src/base-styles/_nx-colors.scss:295]: > In Kaleido, the green is a `green-80`.

Oh nevermind, the color is a `green-35`, but the label says `green-80`. I will let Jenni know.

Comment (rpokorny) [reply lib/src/base-styles/_nx-colors.scss:295]: > Oh nevermind, the color is a green-35, but the label says green-80. I will let Jenni know.

@cindyko226 already asked about that this morning in the channel

**Comment by EmilyBermudez88** [comment]
Is there a way to adjust the tooltip colors? Screenshot 2023-05-01 at 8 58 21 AM

**Comment by EmilyBermudez88** [comment]
`--nx-color-chart-data-8` is labelled as `teal-90` but the actual swatch in the mocks is `teal-80`. Check with design, but I think the label may be wrong since `teal-90` looks quite similar to `grey-97`

**Comment by rpokorny** [comment]
> Is there a way to adjust the tooltip colors?

Yes.  Design will need to provide a color there as what's shown in your mockup clearly isn't acceptable

**Comment by rpokorny** [comment]
> --nx-color-chart-data-8 is labelled as teal-90 but the actual swatch in the mocks is teal-80. Check with design, but I think the label may be wrong since teal-90 looks quite similar to grey-97

Changed to teal-80 in [7c43275](https://github.com/sonatype/sonatype-react-shared-components/pull/1094/commits/7c43275cb632d6acc7a615162255dd98d62627ce)

**Review by EmilyBermudez88 — APPROVED** [review]

**Pull request merged**